lib/Image/Color.cpp

```diff
@@ -31,15 +32,15 @@ static Color clampColor(const Color C) {
                std::clamp(C.B, 0.0, 1.0), C.Space);
 }
 
-static Color RGBToXYZ(const Color Old) {
+static Color rgbToXyz(const Color Old) {
```

Comment:
Not sure how clang-tidy decides which things are initialisms vs words, but I'd expect RGB, XYZ, and LAB to stay capitalized. Does it appease the tool if we call these convertRGBToXYZ? If not this is probably fine, but it feels weirdly inconsistent to me.